json-server.py

```diff
@@ -25,6 +31,20 @@ def do_GET(self):
             response_body = get_post(url["pk"])
             return self.response(response_body, status.HTTP_200_SUCCESS.value)
 
+        elif url["requested_resource"] == "users":
+            if "email" in url["query_params"]:
+                requested_email = url["query_params"]["email"][0]
+                check_for_duplicate = find_email_match(requested_email)
+                if "email_exists" in check_for_duplicate:
```

Comment:
Whether "email_exists" is true or false still returns HTTP status 200.

Reply:
With the 200 code, I wanted to indicate that the get request was successful and return the check_for_duplicates variable to the client for use in the react/js code.